tensorboard/plugins/text/text_plugin_test.py

```diff
@@ -386,8 +305,8 @@ def test_text_array_to_html(self):
     d3 = np.array([[['foo', 'bar'], ['zoink', 'zod']], [['FOO', 'BAR'],
                                                         ['ZOINK', 'ZOD']]])
 
-    warning = text_plugin.markdown_and_sanitize(text_plugin.WARNING_TEMPLATE %
-                                                3)
+    warning = plugin_util.markdown_to_safe_html(text_plugin.WARNING_TEMPLATE
```

Comment:
nit: Maybe place the entire argument on the next line and indent it by 4 spaces.